DTS-STN/Alpha-Site
# Feature/137 report a problem api

# Description

[#137 Write Next JS API route for report a problem form submission](https://trello.com/c/q6UGarhN/137-137-write-next-js-api-route-for-report-a-problem-form-submission)

Wrote API route to take form submissions and submit to notify API to send an email to the specified email.

Note there are 4 new environment variables introduced in this PR. All of which have been added to the docker and helm setup as well as teamcity.

`NOTIFY_BASE_API_URL`: The base url for the notify API `https://api.notification.canada.ca`

`NOTIFY_API_KEY`: The notify API key used for authorization of request

`NOTIFY_REPORT_A_PROBLEM_EMAIL`: The email that the message should be sent too 

`NOTIFY_REPORT_A_PROBLEM_TEMPLATE_ID`: The notify template id to use for the email

## Acceptance Criteria

Done when a report a problem form submission results in sending of an email to an inbox with the details 

## Test Instructions

1. You'll need to have a notify account
2. Once you have one let me know and I'll invite you to the project 
3. Generate your api key
4. Create `.env.local` file with the env variables specified above 
5. Add your email to the safelist 
5. Run project and submit report a problem with at least one checkbox checked 
6. Submit form and see if you get an email

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
